etc/spack/defaults/windows/bootstrap-resource.yaml

```diff
@@ -0,0 +1,16 @@
+bootstrap-resource:
+  # If set to false Spack will not fetch bootstrap resources but instead
+  # proceed without fetching said resources
+  enable: true
+  # Trusted sources to acquire bootstrap resources from
+  resources:
+    gpg:
+      providers:
+        - name: Github Actions
```

Comment:
(question) was Github Actions generating these? What action triggers that? Either way, I think this could be clearer, since it would be the github actions of another repo (https://github.com/spack/windows-bootstrap-resources) that generates these, so it may be more informative to mention that this comes from a Github-hosted repo.

Reply:
A github actions workflow is yes, on push to develop in that repo you linked, the binaries are regenerated and the old binaries are replaced on the pages instance serving them.

Reply:
Renamed to "Spack Bootstrap Resource Repo" and since the url beneath includes the repo I think that covers our bases.